docker/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Grafana Adaptive Metrics Auto-apply (Apply Recommendations)'
+description: 'Apply your Adaptive Metrics recommendations.'
+runs:
+    using: 'docker'
+    image: 'Dockerfile'
+inputs:
+  working-dir:
+    default: './'
+    description: 'The directory to run the apply in.'
+  dry-run:
+    default: 'false'
+    description: 'Whether to apply the recommendations or just print tf plan output.'
```

Comment:
This description no longer applies, right?

Reply:
[aab6cc6](https://github.com/grafana/adaptive-metrics-autoapply/pull/12/commits/aab6cc625d45da60d4c0ca91e748ea52b32054c5)